src/cmd/set.rs

```diff
@@ -146,6 +146,11 @@ impl Set {
         frame.push_bulk(Bytes::from("set".as_bytes()));
         frame.push_bulk(Bytes::from(self.key.into_bytes()));
         frame.push_bulk(self.value);
+        if let Some(ms) = self.expire {
+            // Add expiration in milliseconds: SET key value PX ms
```

Comment:
Can you please elaborate this comment so it explains that `px` means milliseconds, and perhaps also that `ex` would mean seconds? Currently it is just an arbitrary string if the user is not familiar with this part of the redis protocol.

Reply:
Done.